cdf_fabric_replicator/config.py

```diff
@@ -45,6 +45,7 @@ class SourceConfig:
     data_set_id: str
     event_path: Optional[str] = None
     raw_time_series_path: Optional[str] = None
+    read_batch_size: int = 1000
```

Comment:
Great to see the batch size configurable. it doesn't look like the batch size is documented in the README.

Reply:
Will add to documentation